src/open_inwoner/scss/components/Actions/Actions.scss

```diff
@@ -9,130 +10,230 @@
       display: none;
     }
   }
 
+  // Mobile and desktop filter styling
   .actions__filter-container {
     align-items: center;
-    border-bottom: 1px solid var(--color-gray-light);
     display: none;
     gap: var(--spacing-medium);
     grid-template-columns: 1fr;
     margin-bottom: var(--spacing-medium);
     padding: 8px;
 
+    // Tablet filter styling
     @media (min-width: 768px) {
       display: grid;
       grid-template-columns: 2fr 3fr 2fr 2fr;
       gap: var(--spacing-giant);
     }
 
+    // Styling for the date picker
     .input {
       min-width: auto;
     }
   }
 
+  // List styling
   .actions__list {
     display: flex;
     flex-direction: column;
-    gap: var(--spacing-medium);
+    gap: var(--spacing-large);
   }
 
-  .actions__action {
-    display: grid;
-    grid-template-columns: 1fr 1fr;
-    position: relative;
-    border-bottom: 1px solid var(--color-gray-light);
-    padding-bottom: var(--spacing-large);
+  .action__label {
+    // Dropdown select
+    .dropdown > .button {
+      border: 1px solid var(--color-gray-light);
+      padding: 10px 12px 10px 15px;
+      height: unset;
+      min-width: 128px;
+      justify-content: space-between;
+    }
 
-    @media (min-width: 768px) {
-      grid-template-columns: 1fr 1fr 1fr;
+    // Styling dropdown button
+    .dropdown__content .button {
+      border: none;
     }
   }
 
-  .actions__label {
-    padding: var(--spacing-medium);
-    word-break: normal;
-    overflow-wrap: break-word;
+  // Actions footer.
+  &__footer {
+    .button {
+      margin: 0;
+      padding-left: 0;
+      padding-top: var(--spacing-large);
+    }
+  }
+}
+
+// Action styling
+.action {
+  border: 1px var(--color-gray-light) solid;
+  border-radius: var(--border-radius);
+  display: flex;
+  flex-direction: column;
+  gap: 40px;
+  padding: var(--spacing-large);
 
-    &--big {
-      grid-column: 1 / span 2;
+  &__header {
+    display: flex;
+    gap: var(--spacing-giant);
+    flex-direction: column;
+
+    section {
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+    }
+
+    aside {
+      width: fit-content;
+      max-width: 100%;
+
+      @media (min-width: 768px) {
+        max-width: 600px;
+      }
+    }
+
+    .utrecht-heading-3 {
+      width: 100%;
     }
 
     .button {
-      border: 1px solid;
-
-      &.actions__status-selector {
-        &--approval {
-          background-color: var(--color-danger-lightest);
-          border-color: var(--color-danger-lightest);
-          color: var(--color-danger);
-        }
-
-        &--open {
-          background-color: var(--color-info-lightest);
-          border-color: var(--color-info-lightest);
-          color: var(--color-info-darker);
-        }
-
-        &--closed {
-          background-color: var(--color-success-lighter);
-          border-color: var(--color-success-lighter);
-          color: var(--color-success);
-        }
+      width: 100%;
```

Comment:
Note that on mobile the titles are not breaking!
Our minimal supported screenwidth is 360px, so most stuff should still be readable on such a screen:

<img width="659" alt="Screenshot 2025-05-26 at 12 47 06" src="https://github.com/user-attachments/assets/cba08e35-e48c-45dd-ae4e-644d93ae418a" />